test/src/api/routes/test-work.js

```diff
@@ -54,33 +54,36 @@ describe('GET /work', () => {
 		const res = await chai.request(app).get(`/work/${aBBID}/aliases`);
 		expect(res.status).to.equal(200);
 		expect(res.body).to.be.an('object');
-		expect(res.body.aliases).to.be.an('array');
 		expect(res.body).to.have.all.keys(
 			'bbid',
 			'aliases'
 		);
+		expect(res.body.aliases).to.be.an('array');
+		expect(res.body.aliases).to.have.lengthOf(1);
 	 });
 
 	 it('should return list of identifiers of work', async function () {
 		const res = await chai.request(app).get(`/work/${aBBID}/identifiers`);
 		expect(res.status).to.equal(200);
 		expect(res.body).to.be.an('object');
-		expect(res.body.identifiers).to.be.an('array');
 		expect(res.body).to.have.all.keys(
 			'bbid',
 			'identifiers'
 		);
+		expect(res.body.identifiers).to.be.an('array');
+		expect(res.body.identifiers).to.have.lengthOf(1);
```

Comment:
Length of identifiers, aliases, and relationships array should be 1 if the entity inserted successfully. But it is 0 at the time of running tests, This may happen because of some mistake to insert an entity in create-entity in the test database. Endpoint works fine for the main database. Please help to pass these tests. To check this, run only test-work.js. 
`npm test test/src/api/routes/test-work.js`